OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -139,6 +148,9 @@ public class BuildingCategories
 		[Desc("Minimum distance in cells from center of the base when checking for building placement.")]
 		public readonly int MinBaseRadius = 2;
 
+		[Desc("Minimum distance in cells from center of the base when checking for fragile building placement. Make this larger than MinBaseRadius to push fragile structures away from the CY so that they don't get killed with one nuke strike.")]
```

Comment:
> [SP2100] Code line must be no longer than 180 characters (now 245).

Looks like Travis find this comment too long. I think "Make this larger than MinBaseRadius to push fragile structures away from the CY so that they don't get killed with one nuke strike." is unnecessary too.